src/components/bluebutton.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './bluebutton.scss';
+
+const Button = ({ targetTo, label, ...props }) =>
+(
+  <div
+    className={styles.our_button}
+    role="button"
+    tabIndex="-1"
+    {...props}
+    onClick={(event) => {
+      event.preventDefault();
```

Comment:
I don't think we need `preventDefault` here. We can do something like `onClick={()=>onClick()}` or even `onClick={onClick}`.

@banciur Do you know what's best practice here?